### PR TITLE
Added port: The Legend of Zelda: Dungeons of Infinity.

### PR DESCRIPTION
**Pull request by t0b10-r3tr0**
# New Port for The Legend of Zelda: Dungeons of Infinity

## Game Information
- **Title**: The Legend of Zelda: Dungeons of Infinity
- **URL**: https://justinbohemier.wixsite.com/portfolio/game-design

### CFW Tests
Game has been tested on the following major CFWs:
- [ ] AmberELEC
- [ ] ArkOS
- [X] ROCKNIX
- [ ] muOS
- [X] Batocera

### Resolution Tests
Test all major resolutions:
- [ ] 480x320
- [ ] 640x480
- [ ] 720x720 (RGB30)
- [X] Higher resolutions (e.g., 1280x720)

## File Structure
  - zeldadoi/
    - port.json
    - README.md
    - splash.png
    - gameinfo.xml
    - Zelda Dungeons of Infinity.sh
    - zeldadoi/
      - zeldadoi.port
      - gmloader.json
      - gmloader.aarch64
      - lib/
      - license/
      - savedata/
      - tools/

## Additional Resources

Port was tested in the following thread: https://discord.com/channels/1122861252088172575/1309681248003817544

### CODE REVIEW

File: ports/zeldadoi/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+|Select|Game Menu|
+|D-Pad|Movement|
+|Left Analog|Movement|
+
+
+## Compile
+
+N/A.
+
+This port uses GMLoader Next.
```
Comment (JeodC) [inline ports/zeldadoi/README.md:24]:
```suggestion
## Thanks
Justin Bohemier - The game and distribution permission  
JohnnyOnFlame, PortMaster Team - GMLoader  
t0bi0, Shallow - Communications medium  
Jeod, Jantrueno - Port testing and structure  
```

File: ports/zeldadoi/gameinfo.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+ <?xml version="1.0"?>
+<gameList>
+	<game>
+		<path>./Zelda Dungeons of Infinity.sh</path>
+		<name>The Legend of Zelda: Dungeons of Inifity</name>
+		<desc>The Legend of Zelda: Dungeons of Infinity is the result of four years of development in Gamemaker Studio from its creator Justin Bohemier. The game makes use of a number of A Link to the Past's original assets to really capture the same aesthetic, with new features and assets added to support the new procedurally-generated dungeon crawling.</desc>
+		<image>./zeldadoi/splash.png</image>
```
Comment (JeodC) [inline ports/zeldadoi/gameinfo.xml:7]: Should use a `cover.png` and preferably a mixv1 style to remain consistent with other ports. Port is also missing `screenshot.png`. This `splash.png` file is used as a brief loading screen.

File: ports/zeldadoi/zeldadoi/savedata/.gitkeep
Comment (JeodC) [inline ports/zeldadoi/zeldadoi/savedata/.gitkeep:1]: This file and directory is not actually needed, GMLoader and GameMaker will create it on first boot.

**Review by JeodC — CHANGES_REQUESTED** [review]

File: ports/zeldadoi/gameinfo.xml (hunk 2 above)
Comment (JeodC) [reply ports/zeldadoi/gameinfo.xml:7]: zelda-doi

ICYMI

Comment (t0b10-r3tr0) [reply ports/zeldadoi/gameinfo.xml:7]: Thanks. Was wondering about this and just assumed splash could be used for both. I will break out into separate files.

**Comment by JeodC** [comment]
This will also need to go to multiverse once changes are made: https://github.com/PortsMaster-MV/PortMaster-MV-New

**Comment by t0b10-r3tr0** [comment]
Thank you. Resubmitting to the MV repo very soon.

**Pull request closed without merging**